fvp: update Trusted Services to v1.3.0 [was: update TF-A to v2.14]

gyuri-szing opened the pull request:
Update TF-A to latest release and its dependency MbedTLS to v3.6.5. Both projects use git submodules so enable the relevant sync-s option.

odeprez commented on fvp.xml:
```diff
@@ -8,7 +8,7 @@
         <project path="hafnium"              name="hafnium/hafnium.git"                   revision="a33eca9976006ac9b08ed4afe6260a2e8b9d2b3a" clone-depth="1" remote="tfo" />
-        <project path="trusted-firmware-a"   name="TF-A/trusted-firmware-a.git"           revision="refs/tags/v2.11" clone-depth="1" remote="tfo" />
-        <project path="mbedtls"              name="Mbed-TLS/mbedtls.git"                  revision="refs/tags/mbedtls-3.6.0" />
+        <project path="trusted-firmware-a"   name="TF-A/trusted-firmware-a.git"           revision="refs/tags/v2.14" clone-depth="1" remote="tfo"  sync-s="true" />
```
Hi TF-A uses x.y.z version moving forward so should be v2.14.0

gyuri-szing committed:
fvp-ts: update Trusted Services

Update TS to the latest release and remove the TS-specific pinning of
the Linux kernel version, which is no longer needed.

Signed-off-by: Gyorgy Szing <gyorgy.szing@arm.com>
Acked-by: Jens Wiklander <jens.wiklander@linaro.org>

jenswi-linaro commented:
Please remove the `Change-Id:`. With that fixed:
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

gyuri-szing commented:
Thanks, applied!
